CIScripts/Test/Tests/Protocol/SingleHostWithUtils.Tests.ps1

```diff
@@ -130,15 +137,26 @@ Describe "Single compute node protocol tests with utils" {
     }
 
     AfterEach {
-        Write-Host "Removing containers"
-        if (Get-Variable Container1ID -ErrorAction SilentlyContinue) {
-            Invoke-Command -Session $Session -ScriptBlock { docker rm -f $Using:Container1ID } | Out-Null
-            Invoke-Command -Session $Session -ScriptBlock { docker rm -f $Using:Container2ID } | Out-Null
+        Write-Host "Removing container 1"
+        if (Get-Variable "Container1ID" -ErrorAction SilentlyContinue) {
+            Invoke-NativeCommand -Session $Session -CaptureOutput {
+                docker rm -f $Using:Container1ID
+            }
```

Comment:
We think that you should remove -CaptureOutput if you would like to print out result of `docker rm` or add `| Out-Null` after `Invoke-NativeCommand` to drop result of command.

Reply:
`| Out-Null` is not needed

Reply:
`Invoke-NativeCommand` will return an object with `Output` field containing the output, which by default goes to the output stream. Do you want to explain why `Out-Null` is not needed here?

Reply:
It does not go to the output stream by default. I don't know why and I don't care why, but it does not.

Reply:
There a difference whether you run it in interactive mode or not.